### PR TITLE
AppVeyor configuration for PR-based continuous integration.

### PR DESCRIPTION
**Pull request by HSAR**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+branches:
+  only:
+    - dev
+    - sschen/ci
```
Comment (bgavrilMS) [inline appveyor.yml]: Is this required?

Comment (HSAR) [inline appveyor.yml]: I've now turned on the setting for "build only if appveyor.yml is in VCS", so no, this can be removed.

File: appveyor.yml [hunk 2]
```diff
@@ -1,8 +1,3 @@
-branches:
-  only:
-    - dev
```
Comment (bgavrilMS) [inline appveyor.yml]: I'd keep the dev branch there

Comment (HSAR) [inline appveyor.yml]: Commits won't be built unless they have an appveyor.yml file checked in, which for the moment is only on dev. We can re-add this if it becomes a problem for any reason, I think.

**Comment by HSAR** [comment]
Added AppVeyor badges to the README.

**Pull request closed without merging**